aiobotocore/endpoint.py

```diff
@@ -51,43 +69,43 @@ def __dir__(self):
         return attrs
 
     def __del__(self):
-        self.close()
+        # callers may not read the Body resulting in an unclosed ClientResponse
+        if self._loop.is_running():
+            asyncio.ensure_future(self.__response.release(), loop=self._loop)
```

Comment:
So socket connection is not useful if all data from buffer has not been fetched, `release()` tries to read all available data in order to save connection and return it to connection pool (session). But since this object is garbage collected anyway, we should drop connection and warn user that he should release/close connection explicitly.